vulnfeeds/cpp/main.go

```diff
@@ -75,84 +76,85 @@ func InScopeGitRepo(repoURL string) bool {
 	return true
 }
 
-func GetRepoDir(repoURL string, repobasedir string) string {
-	parsedURL, err := url.Parse(repoURL)
-	if err != nil {
-		return ""
-	}
-	return path.Join(repobasedir, parsedURL.Hostname(), parsedURL.Path)
-}
-
-// Either clones the repository or returns the one previously cloned.
-func GetRepo(repoURL string, repobasedir string) (r *git.Repository, e error) {
-	repodir := GetRepoDir(repoURL, repobasedir)
-	// TODO: reorder to open, then clone, once behaviour and failure modes understood
-	Logger.Infof("Cloning %s", repoURL)
-	r, err := git.PlainClone(repodir, false, &git.CloneOptions{
-		URL: repoURL,
-	})
-	if err != nil && err != git.ErrRepositoryAlreadyExists {
-		return nil, err
-	}
-	r, err = git.PlainOpen(repodir)
-	if err != nil {
-		return nil, err
-	}
-	return r, nil
-}
-
-// Examines repos and tries to convert version tags to commits
-func GitVersionToCommit(versions cves.VersionInfo, repos []string, repobasedir string) (v cves.VersionInfo, e error) {
+// Examines repos and tries to convert versions to commits by treating them as Git tags.
+// Takes a CVE string (for logging), cves.VersionInfo with AffectedVersions and
```

Comment:
What exactly is this string? a CVE ID?

Reply:
Yes, a CVE ID. Clarified in https://github.com/google/osv.dev/pull/1105/commits/db519e9ea51359703d144ecd26c1f241c8e8d16c